libs/net/services/Actions/Managers/IngestActionManager`.cs

```diff
@@ -65,14 +65,11 @@ public class IngestActionManager<TOptions> : ServiceActionManager<TOptions>, IIn
     /// Verify that the schedule should run the action.
     /// </summary>
     /// <returns></returns>
-    protected override async Task<bool> PreRunAsync()
+    protected override Task<bool> PreRunAsync()
     {
-        // Always fetch the latest version of the ingest and update this manager with it.
-        var ingest = await this.Api.HandleRequestFailure(async () => await this.Api.GetIngestAsync(this.Ingest.Id), this.Options.ReuseIngests, this.Ingest);
```

Comment:
We should already have the latest ingest configuration, there is no need to make this request too.